.storybook/main.ts

```diff
@@ -0,0 +1,19 @@
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  addons: [
+    '@storybook/preset-create-react-app',
+    '@storybook/addon-onboarding',
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@chromatic-com/storybook',
+    '@storybook/addon-interactions',
+  ],
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
+  },
+  staticDirs: ['..\\public'],
```

Comment:
[fcc4393](https://github.com/kakao-tech-campus-2nd-step2/react-gift-react-foundation/pull/30/commits/fcc43936f5510cb64fbf7d9d2f618d666c384889)에서 수정됐다가 [b24eee3](https://github.com/kakao-tech-campus-2nd-step2/react-gift-react-foundation/pull/30/commits/b24eee3c3f6abd51ed1cda57cd0ecce423d2783f) 커밋에서 다시 되돌아왔네요. 추후 다시 반영하는게 좋겠습니다.

```suggestion
  staticDirs: ['../public'],
```